### ISSUE TITLE
Linux Desktop File `TryExec` key missing

### ISSUE THREAD

**Issue by HarshNarayanJha** [zed-industries/zed#15148]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

The Latest release `v0.145.1-1` removes the `TryExec` key from the Linux desktop file at `/usr/share/applications/dev.zed.Zed.desktop` causing app launches such as rofi (or any other DE menu) to not show Zed in the List

Adding `zeditor` back to the TryExec immediately fixes the issue

This is caused by this [commit](https://github.com/zed-industries/zed/commit/0c6105992cc496976e3d27c0884e5b210074a3ca)  which replaces `TryExec=$APP_CLI` with `TryExec=$APP`, causing it to be empty in the release build
in File: `crates/zed/resources/zed.desktop.in`

### Environment

Zed: v0.145.1 (Zed)
OS: Linux Wayland garuda unknown
Memory: 7.1 GiB
Architecture: x86_64

### If applicable, add mockups / screenshots to help explain present your vision of the feature

Before the update
```desktop
[Desktop Entry]
Version=1.0
Type=Application
Name=Zed
GenericName=Text Editor
Comment=A high-performance, multiplayer code editor.
TryExec=zeditor
StartupNotify=true
Exec=zeditor
Icon=zed
Categories=Utility;TextEditor;Development;IDE;
Keywords=zed;
MimeType=text/plain;inode/directory;

[Desktop Action NewWorkspace]
Exec=zeditor --new
Name=Open a new workspace
```

After the update:
```desktop
[Desktop Entry]
Version=1.0
Type=Application
Name=Zed
GenericName=Text Editor
Comment=A high-performance, multiplayer code editor.
TryExec=
StartupNotify=true
Exec=zeditor
Icon=zed
Categories=Utility;TextEditor;Development;IDE;
Keywords=zed;
MimeType=text/plain;inode/directory;

[Desktop Action NewWorkspace]
Exec=zeditor --new
Name=Open a new workspace
```

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary><pre>

</pre></details>

**Comment by HarshNarayanJha** [comment]
Pull request #15149

**Comment by HarshNarayanJha** [comment]
Closing since #15149 merged

**Comment by HarshNarayanJha** [comment]
This isn't fixed in the latest release `v0.146.3`. Seems like the PR wasn't included or something?

**Comment by SomeoneToIgnore** [comment]
> latest release `v0.146.3`

To be correct, this is not the latest release, but the latest stable release.

> or something

So this: if the PRs are merged into `main` without further cherry-picking, they get into Zed Preview which becomes Zed Stable next week, giving us a week gap to ensure it's not breaking things.

A bit more details: https://zed.dev/docs/development/releases

**Comment by HarshNarayanJha** [comment]
Now I get it. So new pulls have to wait at least a week to get into the stable release. That's okay for the other PR...

But this issue? I updated, and the key was missing. Wasn't this cherry-picked?

TLDR;
Couldn't launch zed from launcher in the latest stable release

**Comment by SomeoneToIgnore** [comment]
I do see the cherry-pick indeed, so presumably something odd happened.
I am preparing a patch for all releases now-ish, so I can pick that commit again into the next stable and see what happens.

**Comment by HarshNarayanJha** [comment]
Okay! Closing

**Comment by SomeoneToIgnore** [comment]
If you're able (and willing to) to build things from source and test, could you check that https://github.com/zed-industries/zed/tree/v0.146.x has what you want?

I plan to release it in ~8 hours from now.

**Comment by HarshNarayanJha** [comment]
No need to build and, test the source tree contains the correct `TryExec` value in the [desktop file](https://github.com/zed-industries/zed/blob/v0.146.x/crates/zed/resources/zed.desktop.in)
